sdv/constraints/base.py

```diff
@@ -245,30 +245,6 @@ def filter_valid(self, table_data):
 
         return table_data[valid]
 
-    @classmethod
-    def from_dict(cls, constraint_dict):
```

Comment:
This is not used anywhere else right now (and it causes problems for this PR), which is why it's been deleted.